tasks/db/init.ts

```diff
@@ -15,17 +14,23 @@ export default async (testDb?: boolean) => {
   const orderBookRepository = new OrderBookRepository(db.models);
   const p2pRepository = new P2PRepository(db);
 
-  await Promise.all([
-    orderBookRepository.addCurrencies([
-      { id: 'BTC' },
-      { id: 'LTC' },
-      { id: 'ZRX' },
-      { id: 'GNT' },
-    ]),
+  await orderBookRepository.addCurrencies([
+    { id: 'BTC' },
+    { id: 'LTC' },
+    { id: 'ZRX' },
+    { id: 'GNT' },
+  ]);
+
+  await Promise.all([,
     orderBookRepository.addPairs([
       { baseCurrency: 'BTC', quoteCurrency: 'LTC', swapProtocol: SwapProtocol.LND },
       { baseCurrency: 'ZRX', quoteCurrency: 'GNT', swapProtocol: SwapProtocol.RAIDEN },
     ]),
+    p2pRepository.addHosts([
```

Comment:
Do we need the seed peers in the test database? I don't see why but maybe I'm missing something. For one thing we can't rely on these nodes being up for the purposes of testing, so it seems like we'd want to avoid that.

Reply:
We want them in both, `xud` as well as `xud_test`. Our goal is to have them always up for exactly the purpose of testing and bootstrapping `hosts`. Especially useful for external testers.